src/.vuepress/theme/templates/build.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="{{ lang }}">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <meta name="generator" content="VuePress {{ version }}" />
+    <script>
```

Comment:
Note: Dit is gekopieerd van de [build template van het default thema van vuepress](https://github.com/vuepress/vuepress-next/blob/61ee4e4f54c16746c3518f14f27d444ceea130c3/ecosystem/theme-default/templates/build.html#L18-L24)